sphinx/templates/quickstart/conf.py_t

```diff
@@ -112,12 +112,16 @@ html_static_path = ['{{ dot }}static']
 #
 # This is required for the alabaster theme
 # refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
-html_sidebars = {
-    '**': [
-        'relations.html',  # needs 'show_related': True theme option to display
-        'searchbox.html',
-    ]
-}
+if html_theme == 'alabaster':
```

Comment:
What will be happened for non alabaster themes?

Reply:
`html_sidebars` won't be set so it should be left at it's default which should be correct for all the other bundled themes. For any other theme that require a custom sidebar setting, the user will have to update it by itself.